bpython/curtsiesfrontend/interpreter.py

```diff
@@ -43,13 +43,13 @@ class BPythonFormatter(Formatter):
     See the Pygments source for more info; it's pretty
     straightforward."""
 
-    def __init__(self, color_scheme, **options):
+    def __init__(self, color_scheme: Dict, **options: int) -> None:
         self.f_strings = {}
         for k, v in color_scheme.items():
             self.f_strings[k] = f"\x01{v}"
         super().__init__(**options)
 
-    def format(self, tokensource, outfile):
+    def format(self, tokensource: Any, outfile: Any) -> None:
```

Comment:
This kind of annotation isn't very useful, if you're not sure how to find out (feel free to ask!) then this might a good file to skip